.travis.yml

```diff
@@ -1,12 +1,18 @@
 language: julia
 os:
   - linux
-  - osx
 julia:
-  - 1.0
-  - 1
   - nightly
 notifications:
   email: false
+
+script:
+  - julia --project --color=yes -e 'using Pkg;
+                                    Pkg.instantiate();
+                                    Pkg.test(coverage=true);
+                                    Pkg.add(["Colors", "ImageCore"]);
```

Comment:
😕 I'm really confused about why it errors in CI but not locally.

https://travis-ci.org/github/JuliaGraphics/ColorTypes.jl/builds/676029217#L269

Reply:
Try deleting Manifest.toml before running the test locally.
The CI runs in the environment of "ColorTypes" Project.toml. Since the environment is just for ColorTypes itself, Manifest.toml does not have the "ColorTypes" entry.

Reply:
This is how I did locally, which I believe is a clean build.

```bash
git clone https://github.com/JuliaGraphics/ColorTypes.jl.git
cd ColorTypes.jl
JULIA_DEPOT_PATH=~/julia_temp julia --project=. --color=yes -e 'using Pkg; \
                                    Pkg.instantiate(); \
                                    Pkg.test(coverage=true); \
                                    Pkg.add(["Colors", "ImageCore"]);
```

Reply:
What version of Julia are you using? Pkg.jl has changed at least in the past month.

Reply:
cf https://github.com/JuliaLang/Pkg.jl/pull/1696

Adding Colors to the ColorTypes project causes a cyclic dependency, although it is only a test dependency.

Reply:
Correct, I can now reproduce it with Julia 1.5-DEV but not with Julia 1.4. Thanks for the link.